scan.go

```diff
@@ -429,6 +429,11 @@ func internalStartScan() error {
 	}
 
 	if syncReport {
+		if version == "" {
+			return fmt.Errorf("version is required for sync report, install vet using supported method: " +
+				"https://docs.safedep.io/quickstart/")
+		}
+
```

Comment:
Should we use cobra's pre run checks to fail-fast during creation phase itself, for such checks ?

Reply:
eg.
```go
cmd.PreRunE = func(cmd *cobra.Command, args []string) error {
	if syncReport && version == "" {
		return fmt.Errorf("version is required for sync report, install vet using supported method: " +
			"https://docs.safedep.io/quickstart/")
	}
	return nil
}
```

Reply:
@OmkarPh Good point. I have moved the validation to `PreRunE` cobra hook.